inc/zoo/swar/SWAR.h

```diff
@@ -144,13 +146,13 @@ template<int NBitsMost, int NBitsLeast, typename T = uint64_t> struct SWARWithSu
         return most(pos) >> (LaneBits*pos)>> NBitsLeast;
     }
 
-    // Blits most sig bits into least significant bits. Experimental.
-    constexpr auto flattenMostToLeast(u32 pos) const noexcept {
+    // Shifts most sig bits into least significant bits. Experimental.
```

Comment:
Because "SIGN" and "SIGNificant" have the prefix "SIG", I'd not abbreviate, but "most significant"

Reply:
improved. Clean up some of the 80 column stuff also.